pkg/controller/migmigration/task.go

```diff
@@ -221,12 +225,12 @@ var FinalItinerary = Itinerary{
 		{Name: WaitForDirectVolumeMigrationToComplete, Step: StepDirectVolume, all: DirectVolume | EnableVolume},
 		{Name: EnsureAnnotationsDeleted, Step: StepRestore, all: HasStageBackup},
 		{Name: EnsureInitialBackupReplicated, Step: StepRestore},
-		{Name: PostBackupHooks, Step: StepRestore},
-		{Name: PreRestoreHooks, Step: StepRestore},
+		{Name: PostBackupHooks, Step: PostBackupHooks, all: HasPostBackupHooks},
```

Comment:
@pranavgaikwad doesn't it break the pipeline to have a step in the middle of another one like this?